src/sentry/grouping/enhancer/__init__.py

```diff
@@ -264,18 +270,20 @@ def _to_config_structure(self):
         ]
 
     def dumps(self):
-        return (
-            base64.urlsafe_b64encode(zlib.compress(msgpack.dumps(self._to_config_structure())))
-            .decode("ascii")
-            .strip("=")
-        )
+        encoded = msgpack.dumps(self._to_config_structure())
 
-    def iter_rules(self):
-        for base in self.bases:
-            base = ENHANCEMENT_BASES.get(base)
-            if base:
-                yield from base.iter_rules()
-        yield from self.rules
+        try:
+            # I don’t want to put DB access into all of the tests ;-)
+            use_zstd = options.get("enhancers.use-zstd")
```

Comment:
Would we want to roll it out to specific orgs? are you confident enough that it will work across the board?

Reply:
I think rolling it out all at once should be fine. I did this with the attachments cache where we saw quite a bit improvement by such a small change. Although I expect this to be far less visible here.